geniza/corpus/management/commands/export_metadata.py

```diff
@@ -1,30 +1,189 @@
+        if not (self._remote_url or hasattr(settings, self.remote_url_key)):
+            raise CommandError(
+                f"Please set {self.remote_url_key} in local_settings.py or settings file."
+            )
+        return self._remote_url or getattr(settings, self.remote_url_key)
+
+    @cached_property
+    def path_data(self):
+        odir = os.path.join(self.local_path, self.repo_dir_data)
+        if not os.path.exists(odir):
+            os.makedirs(odir)
+        return odir
+
+    def get_path_csv(self, docname):
+        return os.path.join(self.path_data, docname + self.ext_csv)
+
+    @cached_property
+    def path_documents_csv(self):
+        return self.get_path_csv("documents")
+
+    @cached_property
+    def path_footnotes_csv(self):
+        return self.get_path_csv("footnotes")
+
+    @cached_property
+    def path_sources_csv(self):
+        return self.get_path_csv("sources")
+
+    @cached_property
+    def path_fragments_csv(self):
+        return self.get_path_csv("fragments")
+
+    @cached_property
+    def paths(self):
+        return [
+            self.path_documents_csv,
+            self.path_footnotes_csv,
+            self.path_sources_csv,
+            self.path_fragments_csv,
+        ]
+
+    def add_and_commit(self):
+        for fn in self.paths:
+            if os.path.exists(fn):
+                self.repo.index.add(fn)
+        self.repo.index.commit(
+            "Auto-syncing @ " + timezone.now().strftime("%Y-%m-%d %H:%M:%S")
+        )
+
+    def write_local(self):
+        # write docs
+
+        with self.timer("Exporting document objects"):
+            PublicDocumentExporter(progress=self.progress).write_export_data_csv(
+                self.path_documents_csv
+            )
+
+        # write sources
+        with self.timer("Exporting source objects"):
+            SourceExporter(progress=self.progress).write_export_data_csv(
+                self.path_sources_csv
+            )
+
+        # write footnotes
+        with self.timer("Exporting footnote objects"):
+            FootnoteExporter(progress=self.progress).write_export_data_csv(
+                self.path_footnotes_csv
+            )
+
+    def sync_remote(self):
+        """Sync local repository content with origin repository. Assumes
+        :meth:`setup_repo` has already been run, and any new or modified
+        files have been committed."""
+        try:
+            with self.timer("Pulling repository"):
+                origin = self.repo.remote(name="origin")
+                # pull any remote changes since our last commit
+                origin.pull()
+
+            # commit?
+            with self.timer("Adding any changes"):
+                self.add_and_commit()
+
+            # push data updates
+            with self.timer("Pushing any changes"):
+                result = origin.push()
+                if result:
+                    info = result[0]
+                    self.print(f"Result = {info.flags}")
```

Comment:
what is this number?

Reply:
This is a numeric flag indicating the push result. I was printing for my own purposes, we can hide